chapters/problems/sections/spherical_problems/subsections/teleportation.tex

```diff
@@ -0,0 +1,6 @@
+\subsection{Teleportation}
+The teleportation mechanism described in \autoref{sub:teleportation} is far from a perfect solution.
+When a model crosses the boundary of one region it has to be physically moved to a completely different position and its velocity has to be altered as well.
+This causes a visible discontinuity in the object's movement.
+The problem is even more apparent when the camera is being teleported.
+Positions of the objects in the scene appear to be slightly different after the teleportation which causes an unpleasant impression of a sudden "jump".
```

Comment:
maybe we could provide 2 screenshots one right before and one right after crossing the boundary between the chunks